ledfx/api/integration_qlc.py

```diff
@@ -49,7 +48,7 @@ async def get(self, integration_id) -> web.Response:
         response["event_types"] = {
             Event.EFFECT_SET: {
                 "event_name": "Effect Set",
-                "event_filters": {"effect_name": effect_names},
+                "event_filters": {"effect_id": effect_ids},
```

Comment:
This is what leads me to ask, it looks like it is a change in the externalized response structure, do you know that to be functional or passive the client.